main_test.go

```diff
@@ -119,6 +120,25 @@ func TestParseAndRunConfig(t *testing.T) {
 			planExpectedOk:       true,
 		},
 
+		// Test exposing secret as environment variable
+		{
+			cfgExpectedOk:        true,
+			env:                  map[string]string{"PLUGIN_ACTION": "deploy", "PLUGIN_TOKEN": validGCPKey, "PLUGIN_SECRETS": `{"var_1":"secretname:latest"}`, "PLUGIN_SERVICE": "my-service", "PLUGIN_IMAGE": "my-image"},
+			cfgExpectedProjectId: "my-project-id",
+			cfgExpectedSecrets:   map[string]string{"var_1": "secretname:latest"},
+			planExpectedFlags:    []string{"--set-secrets", "^:||:^var_1=secretname:latest"},
+			planExpectedOk:       true,
+		},
+		// Test exposing secret as file
+		{
+			cfgExpectedOk:        true,
+			env:                  map[string]string{"PLUGIN_ACTION": "deploy", "PLUGIN_TOKEN": validGCPKey, "PLUGIN_SECRETS": `{"var_1":"/mnt/path:secretname"}`, "PLUGIN_SERVICE": "my-service", "PLUGIN_IMAGE": "my-image"},
```

Comment:
Is this correct?  --> `"var_1":"/mnt/path:secretname"`

Shouldn't this be `"/mnt/path" : "secretname:latest"` ?

Reply:
You're right, thank you!